README.md

```diff
@@ -193,3 +193,11 @@ If you want to remove the curator job, just run:
 kubectl delete cronjob curator
 kubectl delete configmap curator-config
 ``` 
+
+## FAQ
+### Why does `NUMBER_OF_MASTERS` differ from number of master-replicas?
+The environment variables means you need a minimum of 2. If you have 3 masters and one dies, the cluster still works. Minimum master nodes are usually `n/2 + 1`, where `n` is the number of master nodes in a cluster. If you have 5 master nodes, you should have a minimum of 3, less than that and the cluster _stops_. More info: https://www.elastic.co/guide/en/elasticsearch/guide/1.x/_important_configuration_changes.html#_minimum_master_nodes
```

Comment:
_The default value for this environment variable is 2, meaning a cluster will need a minimum of 2 master nodes to operate._

Also, add the following:
_If you scale the number of masters, make sure to update the minimum number of master nodes through the Elasticsearch API as setting environment variable will only work on cluster setup._